packages/calling-stateful-client/src/CallDeclarative.ts

```diff
@@ -5,6 +5,11 @@ import { Call, CallApiFeature, CallFeatureFactoryType } from '@azure/communicati
 import { CallContext } from './CallContext';
 import { transferCallFeatureDeclaratify } from './TransferCallFeatureDeclarative';
 
+/**
+ * TODO: This should likely be exported?
```

Comment:
Can we resolve this todo now? @PorterNan maybe for input here

Reply:
Let's discuss it here, but will do a follow up PR for the change (this one is big enough as it is).